notebooks/279-photo-maker/279-photo-maker.ipynb

```diff
@@ -0,0 +1,1029 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    id_encoder, text_encoder, unet, vae = None, None, None, None
</pre><p>Stable Diffusion XL is quit heavy model itself, for saving memory I can suggest to check that all models already converted and load original pytorch model only in case if not. It helps to avoid extra memory usageand time  if you want to use this notebook as demo when all models already converted</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1632/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
did you mean tokenizer_2 ? Thanks for your reminder, I missed it